### ISSUE TITLE
The models turned red when they were working.

### ISSUE THREAD

**Issue by dok2zino** [iperov/DeepFaceLab#321]
When src.dst's work was in 0.0151, it suddenly changed to 3.1111 and the face of the work turned red

Why did this happen?

And SAE_encoder.h5, Can you recycle SAE_data.dat? Tell me how

**Comment by Auroir** [comment]
This is referred to as "collapsing". Your model was so well tuned at .0151 that when it tried to make it even better the entire house of cards came crashing down. Be very careful with your parameters when your model has loss values this low.

Enable gradient clipping to avoid this. Things like pixel loss (which it looks like you were using) and high style values can be direct causes. Others include bad face sets, Src/Dst being too dissimilar, and simple bad luck. 


As an aside, in my experience if you are still dissatisfied with the results once you reach .0151 then something else is wrong and training isn't going to resolve it. The model is already telling you that it's nearly perfect, if you disagree then something isn't right.

**Comment by Auroir** [comment]
`SAE_data.dat` You can never reuse for a new model, that is the file defines what kind of model you are running and its dimensions. So if you change anything by hand then reuse it, the values are going to be wrong.
The other .h5 files is your actual model data. Technically yes you can reuse them if you set up your new model with the right dimensions then swap out a piece. I don't think this is going to be useful however because it's trained with data coming in looking like X then spitting it out like Y, but X & Y are determined by the other components which will be different now. I've tried it and I didn't think it was worth the time. I just pretrain a model at the size I want and then reuse the whole thing.
